### PR TITLE
Dates when event cannot be created

### PR DESCRIPTION
**Pull request by Pitcity**
now dates when event cannot be created due to non-existing dates is shown to user

### CODE REVIEW

File: src/main/webapp/resources/js/error.js [hunk 1]
```diff
@@ -13,3 +13,27 @@ function callErrorDialog(errorText) {
         show: 500
     });
 }
+
+function eventsWereNotCreated(datesWhenNotCreated) {
+
+    var contentForDialog = '<div>' + messages.event.errors.cannotCreateEventsForNonExistingDates +
+        '</div><br>';
+    datesWhenNotCreated.forEach(function (item) {
+        contentForDialog += item + '<br>'
```
Comment (andryspo) [inline src/main/webapp/resources/js/error.js]: missed ;

**Comment by hrybs** [comment]
deIlta? Who is she?

**Comment by ValeryGutor** [comment]
She is me. This is my previous account, forgot to sign out

**Comment by hrybs** [comment]
Ok. But I think this is not good idea to invite two account for reviewing from one person. It's very funny that somebody receive 2 approve from one person.

**Comment by ValeryGutor** [comment]
Yes, I understand this, but I can't change an approve and I am not an only person who approve this. I don't think it is a big problem now. In future I will be more careful.

**Pull request merged**